sonarcheck_analyzer.py

```diff
@@ -10,10 +10,8 @@
 import collections
 import logging
 
-from lookout.sdk import AnalyzerServicer, add_analyzer_to_server
-from lookout.sdk import service_analyzer_pb2
-from lookout.sdk import service_data_pb2_grpc
-from lookout.sdk import service_data_pb2
+from lookout.sdk.pb import AnalyzerServicer, add_analyzer_to_server, DataStub, \
```

Comment:
By reading here it seems that there's no clear style-guide regarding imports. Maybe it could be useful defining it and consider using [isort](https://pypi.org/project/isort/).

Reply:
👍 though I would propose it to ML team first because they mostly write python.
And then we can follow the same guide to have a consistent code.

Reply:
//cc @vmarkovtsev here is an idea for you